core/src/edu/cornell/gdiac/temporary/entity/Note.java

```diff
@@ -27,172 +20,187 @@
 /**
  * Model class for enemy shells.
  */
-public class Note extends GameObject {
-	public static final float descentSpeed = -3.5f;
+public class Note{
+	public static final float descentSpeed = -4.5f;
 	/** Rescale the size of a shell */
 	private static final float SHELL_SIZE_MULTIPLE = 4.0f;
 	/** How fast we change frames (one frame per 4 calls to update) */
 	private static final float ANIMATION_SPEED = 0.25f;
 	/** The number of animation frames in our filmstrip */
 	private static final int   NUM_ANIM_FRAMES = 4;
-
-	/** Friction multiplier for this shell */
-	//private float friction;
-	/** Minimum Y velocity for this sheall */
-	private float minvelocy;
 	/** Current animation frame for this shell */
 	private float animeframe;
-	
-	/** To measure if we are damaged */
-	private boolean damaged;
-	/** The backup texture to use if we are damaged */
-	private Texture dmgTexture;
 
-	public int hitStatus;
+	private int hitStatus;
+
+	public int getHitStatus(){
+		return hitStatus;
+	}
+	public void setHitStatus(int t){
+		hitStatus = t;
+	}
 
 	/** line the note is one */
-	public int line;
+	private int line;
+	public int getLine(){
+		return line;
+	}
+	public void setLine(int t){
+		line = t;
+	}
+	private int startFrame;
+	public int getStartFrame(){
+		return startFrame;
+	}
+	public void setStartFrame(int t){
+		startFrame = t;
+	}
+	private int holdFrame;
 
-	public int startFrame;
-	public int holdFrame;
+	public int getHoldFrames(){
+		return holdFrame;
+	}
+	public void setHoldFrames(int t){
+		holdFrame = t;
+	}
 
 	public enum NType{
```

Comment:
`NoteType` but yeah my code doesn't really matter

Reply:
Intended to be changed after this PR

Reply:
I just looked at your NoteType, its the same as NType, but I will change the name